sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -9,23 +9,20 @@
 ### Bugs Fixed
 
 ### Other Changes
+* Python 2.7 and 3.6 are no longer supported. Please use Python version 3.7 or later.
```

Comment:
Please add 3.6 back. :)

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/22564